### ISSUE TITLE
Problem in executeCommand.ts

### ISSUE THREAD

**Issue by SrBrahma** [Duroktar/vscode-npm-scripts#17]
When clicking a script, it was executing "auto run scripName". Reading the code, I found out this:

```ts
const packageManager: string =
      workspace.getConfiguration("npm").get("packageManager") || "npm";
```
In my VsCode configs, in npm>packageManager, it was using the "auto" option, to select automatically if it is npm or yarn.

Got it? I changed it to npm and it is now being executed correctly. The extension should have a way to deal with this.

**Comment by Duroktar** [comment]
Thanks for pointing that out. I'll try to patch it this weekend along with [#16](https://github.com/Duroktar/vscode-npm-scripts/issues/16). Cheers

**Comment by laura-san-martin** [comment]
> When clicking a script, it was executing "auto run scripName". Reading the code, I found out this:
> 
> ```ts
> const packageManager: string =
>       workspace.getConfiguration("npm").get("packageManager") || "npm";
> ```
> 
> In my VsCode configs, in npm>packageManager, it was using the "auto" option, to select automatically if it is npm or yarn.
> 
> Got it? I changed it to npm and it is now being executed correctly. The extension should have a way to deal with this.

For anyone wondering, you can set this in VS Code > File > Preferences > Settings, scroll down till Extensions > NPM and set Package Manager to npm:

![image](https://user-images.githubusercontent.com/14986428/102396224-e72ec100-3fba-11eb-948f-02f1c0000e67.png)

It's a wrokaround for now. 
I believe it was a VS Code update that set this 'auto' thing, mine was working perfectly until a few days ago.

**Comment by SrBrahma** [comment]
> > When clicking a script, it was executing "auto run scripName". Reading the code, I found out this:
> > ```ts
> > const packageManager: string =
> >       workspace.getConfiguration("npm").get("packageManager") || "npm";
> > ```
> > 
> > 
> > In my VsCode configs, in npm>packageManager, it was using the "auto" option, to select automatically if it is npm or yarn.
> > Got it? I changed it to npm and it is now being executed correctly. The extension should have a way to deal with this.
> 
> For anyone wondering, you can set this in VS Code > File > Preferences > Settings, scroll down till Extensions > NPM and set Package Manager to npm:
> 
> ![image](https://user-images.githubusercontent.com/14986428/102396224-e72ec100-3fba-11eb-948f-02f1c0000e67.png)
> 
> It's a wrokaround for now.
> I believe it was a VS Code update that set this 'auto' thing, mine was working perfectly until a few days ago.

Yeah, mine was also working until recently (and then I found out the causing issue).

However, if I remember right, I had already used this extension some time ago and then I removed it as it was throwing the same issue. Maybe it's not the first time VSCode change something about the current package manager checker.
